scripts/github-actions/ga-pause-test.sh

```diff
@@ -23,8 +23,21 @@ source ${REPO_ROOT}/scripts/fetch_ext_bins.sh && \
 	fetch_tools && \
 	setup_envs
 
+if [[ -z "${RUN_TESTS:-}" ]]; then
+  RUN_TESTS=""
+  RUN_TESTS+="TestPauseInSeries/fixtures/iamserviceaccount" # IAM
+  RUN_TESTS+="|TestPauseInSeries/fixtures/logbucketmetric" # Direct
+  RUN_TESTS+="|TestPauseInSeries/fixtures/cloudidsendpoint" # DCL
+  RUN_TESTS+="|TestPauseInSeries/fixtures/computemanagedsslcertificate" # TF
```

Comment:
The challenge is for me/ us to know when a resource is not DCL/ TF anymore. If this is migrated to direct, there's no automated way to know that at the moment. I can gnaw on this issue outside of this PR.